src/adapter.js

```diff
@@ -117,175 +103,171 @@ export function parse({minim, source}, done) {
-          return parameter.name;
-        });
+      // TODO: Handle parameters on a resource level
+      // See https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#path-item-object
+      const relevantPaths = _.omit(pathValue, 'parameters', '$ref');
 
-        resource.attributes.set('href', basePath + href + '{?' + queryParameterNames.join(',') + '}');
-      } else {
-        resource.attributes.set('href', href);
-      }
+      // Each path is an object with methods as properties
+      _.each(relevantPaths, (methodValue, method) => {
+        const methodValueParameters = methodValue.parameters || [];
 
-      const transition = new Transition();
-      resource.content.push(transition);
+        const queryParameters = methodValueParameters.filter((parameter) => {
+          return parameter.in === 'query';
+        });
 
-      // Prefer description over summary since description is more complete.
-      // According to spec, summary SHOULD only be 120 characters
-      const transitionDescription = methodValue.description ?
-        methodValue.description : methodValue.summary;
-      if (transitionDescription) {
-        transition.push(new Copy(transitionDescription));
-      }
+        // URI parameters are for query and path variables
+        const uriParameters = methodValueParameters.filter((parameter) => {
+          return parameter.in === 'query' || parameter.in === 'path';
+        });
 
-      if (methodValue.operationId) {
-        transition.meta.set('title', methodValue.operationId);
-      }
+        // Body parameters are ones that define JSON Schema
+        const bodyParameters = methodValueParameters.filter((parameter) => {
+          return parameter.in === 'body';
+        });
 
-      // For each uriParameter, create an hrefVariable
-      if (uriParameters.length > 0) {
-        transition.hrefVariables = new HrefVariables();
+        // Query parameters are added the HREF if they exist
+        if (queryParameters.length > 0) {
+          const queryParameterNames = queryParameters.map((parameter) => {
+            return parameter.name;
+          });
 
-        uriParameters
-          .map(paramToElement)
-          .forEach((member) => transition.hrefVariables.content.push(member));
-      }
+          resource.attributes.set('href', basePath + href + '{?' + queryParameterNames.join(',') + '}');
+        } else {
+          resource.attributes.set('href', href);
+        }
 
-      // Currently, default responses are not supported in API Description format
-      const relevantResponses = _.omit(methodValue.responses, 'default');
+        const transition = new Transition();
+        resource.content.push(transition);
 
-      if (_.keys(relevantResponses).length === 0) {
-        createTransaction(minim, transition, method);
-      }
+        transition.meta.set('title', methodValue.summary || methodValue.operationId || '');
```

Comment:
@netmilk You OK with falling back to `operationId` here? Or would you rather it be blank?